documentation/asciidoc/accessories/iqaudio/introduction.adoc

```diff
@@ -1,49 +1,32 @@
-== Overview of IQAudio products
+== Overview of Raspberry Pi Audio
+ 
+Introducing our range of boards designed to bring high quality audio to your existing Hi-Fi or Raspberry Pi based equipment and projects.
 
-IQaudio sound cards are all designed to work with the current Raspberry Pi and those that have a
-40way male GPIO pin header. This includes Raspberry Pi 4B, 3B{plus}, 3A{plus}, 3, Zero etc. 
+With 4 different boards to choose from all our audio boards will fit any Raspberry Pi utilising a 40-pin GPIO header.
```

Comment:
This sentence _probably_ isn't necessary, as none of the other HAT pages at https://www.raspberrypi.com/documentation/accessories/ mention that a 40-pin GPIO header is required? Every Raspberry Pi since the B+ has had a 40-pin header, and that was [released](https://www.raspberrypi.com/products/raspberry-pi-1-model-b-plus/) way back in 2014 :wink: